cli/docs/configs/provider.md

```diff
@@ -58,7 +58,7 @@ Logs configuration
 
 | Property         | Type    | Required | Description                     |
 |------------------|---------|----------|---------------------------------|
-| `logLevel`       | string  | No       | Log level. Default: info        |
+| `logLevel`       | string  | No       | Log level. Default: debug       |
```

Comment:
Hm, will it always show debug logs now?

Reply:
@justprosh asked for this to be the default

Reply:
yeah, for first phases of mainnet/testnet i want to see all logs by default in providers' grafanas

Reply:
Even for these?

```
nox::bootstrapper
yamux::connection::stream
tokio_threadpool
tokio_reactor
mio
tokio_io
soketto
yamix
multistream_select
libp2p_swarm
libp2p_secio
libp2p_websocket::framed
libp2p_ping
libp2p_core::upgrade::apply
libp2p_core::upgrade
libp2p_kad::kbucket
libp2p_kad
libp2p_kad::query
libp2p_kad::iterlog
libp2p_plaintext
libp2p_identify::protocol
wasmer_wasi
wasmer_interface_types_fl
polling
walrus
regalloc2
cranelift_wasm
wasmtime_cranelift
tokio
libp2p_noise
yamux
wasmtime_jit
wasi_common
particle_reap
marine_core::module::marine_module
runtime::resource
```